GitUp/Application/Document.m

```diff
@@ -85,7 +77,7 @@ static inline WindowModeID _WindowModeIDFromString(NSString* mode) {
     return kWindowModeID_Stashes;
   }
   XLOG_DEBUG_UNREACHABLE();
-  return nil;
+  return 0;
```

Comment:
Feel like returning a window mode would be more appropriate here (especially if the values changed for some reason), but not really an issue in practice.

Reply:
I had the same thought and the same preference, but I followed the convention of other locations in the project that explicitly return 0 as a sigil for an enum conversion routine.

Reply:
Addressed in next push